crates/accelerate/src/target_transpiler/mod.rs

```
@@ -101,7 +141,11 @@ impl<'py> IntoPyObject<'py> for NormalOperation {
     type Error = PyErr;
 
     fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error> {
-        Ok(self.op_object.bind(py).clone())
+        Ok(self
+            .op_object
+            .get_or_init(|| self.create_py_op(py, None).unwrap())
```

Should we store `op_object` as a `PyResult<PyObject>` instead so we can propagate the error if `create_py_op` fails?

---

I think it might not be necessary, I could just extract the operation before setting it instead.

---

Fixed in 3547921

---

After an internal conversation, we have gone with storing a `PyResult`. Fixed in cd8e63b.